scripts/getDependencies.pl

```diff
@@ -230,18 +230,14 @@ sub downloadFile {
 	my ( $url, $filename ) = @_;
 	print "downloading $url\n";
 	my $output;
-	# ToDo: should use curl only
-	if ($^O eq 'os390') {
-		# Delete existing file in case it is tagged incorrectly which curl would then honour..
-		qx(rm $filename);
-		# .txt SHA files are in ISO8859-1
-		if ('.txt' eq substr $filename, -length('.txt')) {
-			$output = qx{_ENCODE_FILE_NEW=ISO8859-1 curl -k -o $filename $url 2>&1};
-		} else {
-			$output = qx{_ENCODE_FILE_NEW=UNTAGGED curl -k -o $filename $url 2>&1};
-		}
+
+	# Delete existing file in case it is tagged incorrectly which curl would then honour..
+	qx(rm $filename);
```

Comment:
Only need to delete file if it exists. Otherwise, in the Grinder, we got lots errors about `rm: cannot remove`:

```
17:06:16  --------------------------------------------
17:06:16  Starting download third party dependent jars
17:06:16  --------------------------------------------
17:06:16  downloading dependent third party jars to /home/jenkins/workspace/Grinder_TKG/openjdk-tests/TKG/../TKG/lib
17:06:16  downloading https://repo1.maven.org/maven2/org/ow2/asm/asm-all/6.0_BETA/asm-all-6.0_BETA.jar
17:06:16  rm: cannot remove `/home/jenkins/workspace/Grinder_TKG/openjdk-tests/TKG/../TKG/lib/asm-all.jar': No such file or directory
17:06:16  --> file downloaded to /home/jenkins/workspace/Grinder_TKG/openjdk-tests/TKG/../TKG/lib/asm-all.jar
17:06:16  downloading https://repository.ow2.org/nexus/content/repositories/releases/org/ow2/asm/asm/7.3/asm-7.3.jar
17:06:16  rm: cannot remove `/home/jenkins/workspace/Grinder_TKG/openjdk-tests/TKG/../TKG/lib/asm-7.3.jar': No such file or directory
17:06:17  --> file downloaded to /home/jenkins/workspace/Grinder_TKG/openjdk-tests/TKG/../TKG/lib/asm-7.3.jar
17:06:17  downloading https://repo1.maven.org/maven2/commons-cli/commons-cli/1.2/commons-cli-1.2.jar
17:06:17  rm: cannot remove `/home/jenkins/workspace/Grinder_TKG/openjdk-tests/TKG/../TKG/lib/commons-cli.jar': No such file or directory
17:06:17  --> file downloaded to /home/jenkins/workspace/Grinder_TKG/openjdk-tests/TKG/../TKG/lib/commons-cli.jar
17:06:17  downloading https://repo1.maven.org/maven2/org/apache/commons/commons-exec/1.1/commons-exec-1.1.jar
```